quipucords/tests/fingerprinter/test_fingerprint_task.py

```diff
@@ -338,7 +339,7 @@ def _create_satellite_details_report_json(  # noqa: PLR0913, PLR0912, C901
     hostname="9.8.7.6",
     os_name="RHEL",
     os_release="RHEL 7.3",
-    os_version=7.3,
+    os_version="7.3",
```

Comment:
without making this change, tests relying on this helper fail. Would you consider this enough, coverage-wise @infinitewarp ?